### PR TITLE
few fixes

### PR DESCRIPTION
**Pull request by qualidan**
sandbox naming convention
+fix for when stashing an already committed state
+ format of status updates

### CODE REVIEW

File: colony/branch_utils.py [hunk 1]
```diff
@@ -25,6 +25,9 @@ def examine_blueprint_working_branch(repo: BlueprintRepo, blueprint_name: str):
     if repo.is_dirty():
         logger.warning("You have uncommitted changes")
 
+    if repo.untracked_files:
+        logger.warning("You have untracked files")
```
Comment (doppleware) [inline colony/branch_utils.py:29]: Change to "Untracked files detected - only staged or committed files will be used when testing local changes".

File: colony/branch_utils.py [hunk 2]
```diff
@@ -97,7 +100,8 @@ def switch_to_temp_branch(repo: BlueprintRepo, defined_branch_in_file: str) -> s
     uncommitted_branch_name = UNCOMMITTED_BRANCH_NAME + defined_branch_in_file + "-" + random_suffix
     try:
         # todo return id and use it for revert_from_temp_branch
-        stash_local_changes_and_preserve_uncommitted_code(repo)
+        if repo.is_dirty() or repo.untracked_files:
```
Comment (doppleware) [inline colony/branch_utils.py:103]: This looks like a bug no? untracked files are not stashed so we'll run into the same issue. 
We should only stash when we have uncommitted staged code

File: colony/commands/sb.py [hunk 3]
```diff
@@ -171,6 +167,15 @@ def do_start(self):
 
         branch_to_be_used = temp_working_branch or working_branch
 
+        if name is None:
+            suffix = datetime.datetime.now().strftime("%b%d%Y-%H:%M:%S")
```
Comment (doppleware) [inline colony/commands/sb.py:171]: We don't need the year - please remove it

File: colony/branch_utils.py (hunk 2 above)
Comment (qualidan) [reply colony/branch_utils.py:103]: they are stashed:
repo.git.stash("save", "--include-untracked")

Comment (qualidan) [reply colony/branch_utils.py:106]: Now it`s part of logger information

File: colony/branch_utils.py [hunk 4]
```diff
@@ -26,7 +26,7 @@ def examine_blueprint_working_branch(repo: BlueprintRepo, blueprint_name: str):
         logger.warning("You have uncommitted changes")
 
     if repo.untracked_files:
-        logger.warning("You have untracked files")
+        logger.warning("Untracked files detected - only staged or committed files will be used when testing local changes")
```
[the remark this answers is not in the document]
Comment (qualidan) [reply colony/branch_utils.py]: it`s now logger debug info

**Pull request merged**